test/lib/db.spec.ts

```
@@ -0,0 +1,193 @@
+
+    test("adds debug logging in development environment", async () => {
+        // Set environment to development
+        mockProcess.env.NODE_ENV = "development";
+
+        // Import the module to trigger the connection code
+        await import("~/src/lib/db");
+
+        // Verify postgres was called with debug option
+        const mockcalls = vi.mocked(postgres).mock.calls[0];
+        
+        let postgresOptions ;
```

Biome (1.9.4)

[error] This variable implicitly has the any type.

Variable declarations without type annotation and initialization implicitly have the any type. Declare a type or initialize the variable with some value.

(lint/suspicious/noImplicitAnyLet)